### ISSUE TITLE
Important! 

### ISSUE THREAD

**Issue by chelBot** [Xewis2017/f1-3-c2p1-colmar-academy#4]
For improved accessibility, don't nest img tags within p tags. Semantically, p tags should be used for paragraphs. You can use the display property if you want this img to behave as a block element. 
https://github.com/Xewis2017/f1-3-c2p1-colmar-academy/blob/master/CapstoneColmar/index.html#L35
